influxql/parser.go

```diff
@@ -2688,11 +2688,13 @@ func (p *Parser) scan() (tok Token, pos Pos, lit string) { return p.s.Scan() }
 
 // scanIgnoreWhitespace scans the next non-whitespace token.
 func (p *Parser) scanIgnoreWhitespace() (tok Token, pos Pos, lit string) {
```

Comment:
Probably this method name and its comment should reflect that it skips whitespace and comments now.